ui/scripts/inlineHtml/inlineScripts.js

```diff
@@ -1050,11 +1050,11 @@ on('remove:repeating_skill',
     updateTotalCP('skill');
   });
 
-on('clicked:addbaseskills', () => {
+function createBaseSkillsAttributes() {
   const section = 'skill';
   const newAttributes = {};
   function addNewAttributeRow(attributesBySuffix) {
-    const rowId = generateRowID();
+    const rowId = generateTrulyUniqueRowId();
```

Comment:
Turns out that `generateRowID` is not always unique according to the docs...
Added a simple tool to help.